axes/models.py

```diff
@@ -76,6 +76,7 @@ class Meta(object):
         verbose_name = _('access attempt')
         verbose_name_plural = _('access attempts')
 
+
```

Comment:
Just stating that stylistic changes should not be mixed with functional changes; makes tracking regressions and bisecting harder.

Reply:
I agree, shall I remove it from the PR?

Reply:
Either is OK, just making a note :)